code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -73,6 +74,11 @@
 	borgo = null
 	return ..()
 
+/obj/item/modular_computer/tablet/integrated/attackby(obj/item/W as obj, mob/user as mob)
+	if(W.tool_behaviour == TOOL_SCREWDRIVER)
+		return
```

Comment:
Not a hard request, but if this and the parent one could be handled by screwdriver_act, that'd be positive.

Reply:
Done and done